packages/jest-resolve/src/index.ts

```diff
@@ -79,6 +79,10 @@ class Resolver {
     this._modulePathCache = new Map();
   }
 
+  static clearCache() {
```

Comment:
Since this only clears the cache of the default resolver, should the name change to reflect that?

Potentially, add the property to the resolver itself, and check of `resolver.clearCache` or something exists?